src/extension/extension.ts

```diff
@@ -259,7 +260,15 @@ export async function activate(context: vs.ExtensionContext, isRestart: boolean
 	// Fire up Flutter daemon if required.
 	if (workspaceContext.hasAnyFlutterProjects && sdks.flutter) {
 		flutterDaemon = new FlutterDaemon(logger, workspaceContext as FlutterWorkspaceContext, flutterCapabilities);
-		deviceManager = new FlutterDeviceManager(logger, flutterDaemon, config, workspaceContext);
+
+		let runIfNoDevices;
+		if (workspaceContext.config.forceFlutterWorkspace && workspaceContext.config.restartMacDaemonMessage) {
+			runIfNoDevices = () => {
+				const instruction = workspaceContext.config.restartMacDaemonMessage;
+				promptToReloadExtension(`${instruction} (VSCode settings currently expects port: ${config.daemonPort}.)`, `Reopen this workspace`);
```

Comment:
nit: I think this reads better as "VSCode settings currently expect port" (no "s" on "expect"), or "VSCode currently expects port" (no "settings"). VS Code is also usually written with a space or just as "Code". Perhaps just "(Settings current expect port x)" also works and is a bit less verbose.

Reply:
argh i've been writing "VSCode" everywhere haha 😢